### ISSUE TITLE
Provisioning AWS S3 fails because the Ansible module no longer exists

### ISSUE THREAD

**Issue by Dannyb48** [CentOS-PaaS-SIG/linchpin#1262]
**Describe the bug**
I was testing provisioning S3 resource and I got a failure from Ansible that the module being used no longer exists. 

```
TASK [aws : Provisioning AWS_S3 Resource] *************************************************************************************************************************************************************************

2019-08-07 21:29:02,593 INFO fatal: [localhost]: FAILED! => {"changed": false, "msg": "This module has been removed.  The module documentation for Ansible-2.6 may contain hints for porting"}
```
Looking at the playbook its using the **s3** module but I don't see that module listed in the Ansible Cloud Module index. It seems like the module Linchpin should be using is **aws_s3**

Using the following versions of Ansible and Linchpin

```
$ pip freeze | grep ansible
ansible==2.8.2
ansible-lint==4.1.0

$ pip freeze | grep linchpin
linchpin==1.7.6.2
```

**To Reproduce**
1. Create a simple pinfile
```
aws-s3-new:
  topology:
    topology_name: aws-s3-new
    resource_groups:
      - resource_group_name: "aws_s3"
        resource_group_type: "aws"
        resource_definitions:
          - name: "s3_bucket"
            role: "aws_s3"
            region: "{{ region | default('us-east-1') }}"
        {% if credentials is defined %}
        credentials:
          filename: {{ credentials.filename }}
          profile: {{ credentials.profile }}
        {% else %}
        credentials:
          filename: aws.key
          profile: default
        {% endif %}
```
2. run ```linchpin up```
3. See error

**Expected behavior**
When specifying the aws_s3 role a bucket is created on AWS

**Comment by samvarankashyap** [comment]
I see that ansible 2.8.2 is no longer compatible with aws_s3 module, 
We need to replace that module with the newer version 
https://docs.ansible.com/ansible/latest/modules/aws_s3_module.html#aws-s3-module
The existing module needs to be replaced with the latest version in both provision and teardown playbooks.

**Comment by samvarankashyap** [comment]
@Dannyb48 We will try to include fixes to both bugs #1261 #1262 in the upcoming release. 
Thank you for finding it out.

**Comment by Dannyb48** [comment]
Awesome, @samvarankashyap thank you!

If you feel like you guys need help let me know, especially with any of the issues I've filed.
